cmd/clusterctl/internal/test/fake_proxy.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"errors"
 
+	addonsv1 "sigs.k8s.io/cluster-api/api/addons/v1beta1"
```

Comment:
Let's move this import down to the CAPI dependencies section (linter doesn't like it here)